docs/server/source/cloud-deployment-templates/node-on-kubernetes.rst

```diff
@@ -159,36 +164,102 @@ but it should become "Bound" fairly quickly.
    No events.
 
 
-Step 5: Deploy MongoDB & BigchainDB
------------------------------------
+Now we are ready to run MongoDB and BigchainDB on our Kubernetes cluster.
 
-Now you can deploy MongoDB and BigchainDB to your Kubernetes cluster.
-Currently, the way we do that is we create a StatefulSet with two
-containers: BigchainDB and MongoDB. (In the future, we'll put them
-in separate pods, and we'll ensure those pods are in different nodes.)
-We expose BigchainDB's port 9984 (the HTTP API port)
-and MongoDB's port 27017 using a Kubernetes Service.
+Step 5: Run MongoDB as a StatefulSet
+------------------------------------
 
-Get the file ``node-mdb-ss.yaml`` from GitHub using:
+Get the file ``mongo-ss.yaml`` from GitHub using:
 
 .. code:: bash
 
-   $ wget https://raw.githubusercontent.com/bigchaindb/bigchaindb/master/k8s/node-mdb-ss.yaml
+   $ wget https://raw.githubusercontent.com/bigchaindb/bigchaindb/master/k8s/mongodb/mongo-ss.yaml
+
+
+Note how the MongoDB container uses the ``mongo-db-claim`` and the
+``mongo-configdb-claim`` PersistentVolumeClaims for its ``/data/db`` and
+``/data/configdb`` diretories (mount path).
+
+Note also that we use the pod's ``securityContext.capabilities.add`` specification
```

Comment:
This sentence and the next few are about the same thing so should be combined into one paragraph: just remove the blank space between them.